Subtester/Assets/Scenes/Main.unity

```diff
@@ -204,6 +204,50 @@ CanvasRenderer:
   m_PrefabAsset: {fileID: 0}
   m_GameObject: {fileID: 53425789}
   m_CullTransparentMesh: 0
+--- !u!1 &300254081
```

Comment:
I created a separate GameObject to hold the `DeepLinkListener` script. Reason is that we are making it a singleton and our existing object isn't, so I just left the existing object as is and just separated the logic into this separate GameObject. Now the scene looks like this:
<img width="283" alt="image" src="https://github.com/user-attachments/assets/59386b19-f0cd-4c0b-b802-ecbcefd20462" />